app/controllers/students_controller.rb

```diff
@@ -135,6 +135,22 @@ def update_settings
     redirect_to action: 'settings'
   end
 
+  def destroy
+    @role = record
+    begin
+      @role.destroy!
+    rescue ActiveRecord::DeleteRestrictionError => e
+      flash_message(:error, I18n.t('flash.students.destroy.restricted', user_name: @role.user_name, message: e.message))
+      head :conflict
+    rescue ActiveRecord::RecordNotDestroyed => e
+      flash_message(:error, I18n.t('flash.students.destroy.error', user_name: @role.user_name, message: e.message))
+      head :bad_request
+    else
+      flash_now(:success, I18n.t('flash.students.destroy.success', user_name: @role.user_name))
+      head :ok
```

Comment:
`:no_content` is more appropriate here (since no content is being returned). See https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/DELETE.

I'm not sure we did this for the TA destroy, but if not you can switch that one too!